lib/resources/submissions.js

```diff
@@ -211,11 +213,40 @@ module.exports = (service, endpoint) => {
           );
         }));
 
-    service.get(`${base}.csv.zip`, endpoint((container, { params, auth, query }, _, response) =>
-      csvzip(container, auth, params, query, response)));
+    const csv = ({ Key, Form, SubmissionDef }, auth, params, passphrases, response, rootOnly) =>
+      getForm(params, Form)
+        .then((form) => auth.canOrReject('submission.read', form))
+        .then((form) => Promise.all([
+          form.def.getFields(),
+          SubmissionDef.streamForExport(form.id, draft, Object.keys(passphrases)),
+          Key.getDecryptor(map(decodeURIComponent, passphrases)) // will do nothing if {}
```

Comment:
This is making me think of #289. I think Express will automatically decode passphrase query parameters?